### PR TITLE
Update changes-ee-5.3.md for e5.3.2

### PR DESCRIPTION
**Pull request by yanzhiemq**

### CODE REVIEW

**Review by id — DISMISSED** [review]

**Review by hjianbo — DISMISSED** [review]

File: en_US/changes/changes-ee-5.3.md [hunk 1]
```diff
@@ -1,5 +1,58 @@
...
 
+## e5.3.2
+
+## Enhancements
+
+- [#11752](https://github.com/emqx/emqx/pull/11752) Change default RPC driver from 'gen_rpc' to 'rpc' for core-replica database sync.
+
+  This improves core-replica data replication latency.
+
+- [#11785](https://github.com/emqx/emqx/pull/11785) Allow viewer to change their own passwords, viewer can't change other's password.
```
Comment (zmstone) [inline en_US/changes/changes-ee-5.3.md]:
```suggestion
- [#11785](https://github.com/emqx/emqx/pull/11785) Allow `viewer` dashboard role to change their passwords.
```

File: en_US/changes/changes-ee-5.3.md [hunk 2]
```diff
@@ -1,5 +1,58 @@
...
+
+- [#11752](https://github.com/emqx/emqx/pull/11752) Change default RPC driver from 'gen_rpc' to 'rpc' for core-replica database sync.
+
+  This improves core-replica data replication latency.
+
+- [#11785](https://github.com/emqx/emqx/pull/11785) Allow `viewer` dashboard role to change their passwords.
+
+- [#11787](https://github.com/emqx/emqx/pull/11787) Improve `emqx` command performance.
+
+  Avoid loading EMQX application code in `nodetool` script unless necessary.
```
Comment (wivwiv) [inline en_US/changes/changes-ee-5.3.md]: This is for internal developers, not for users
```suggestion
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

File: zh_CN/changes/changes-ee-5.3.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
 
+## e5.3.2
+
+## 增强
+
+- [#11752](https://github.com/emqx/emqx/pull/11752) 将 core-replica 数据库同步的默认 RPC 驱动从 `gen_rpc` 更改为 `rpc`。
+
+  这提升了核心副本数据复制的速度。
+
+- [#11785](https://github.com/emqx/emqx/pull/11785) 允许拥有“查看者”角色的用户更改自己的密码。然而，拥有“查看者”角色的用户没有权限更改其他用户的密码。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11785](https://github.com/emqx/emqx/pull/11785) 拥有“查看者”角色的用户具有更改自己密码的权限，但无权更改其他用户密码。
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 4]
```diff
@@ -1,5 +1,47 @@
...
+
+- [#11787](https://github.com/emqx/emqx/pull/11787) 提升了 `emqx` 命令的性能。
+
+- [#11790](https://github.com/emqx/emqx/pull/11790) 为配置在 Redis 授权数据源中的 Redis 命令添加了验证功能。此外，此次改进优化了认证和授权过程中 Redis 命令的解析。现在的解析符合 `redis-cli` 兼容性标准，并支持引号参数。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11790](https://github.com/emqx/emqx/pull/11790) 为 Redis 授权数据源中的 Redis 命令添加了验证功能。此外，此次改进优化了认证和授权过程中 Redis 命令的解析，现在的解析符合 `redis-cli` 兼容性标准，并支持引号参数。
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 5]
```diff
@@ -1,5 +1,47 @@
...
+
+- [#11541](https://github.com/emqx/emqx/pull/11541) 通过引入更多的互动方式，增强了 EMQX 的文件传输能力。现在，客户端可以将文件传输命令发送到 `$file-async/...` 主题，而不是 `$file/...`。命令执行结果将作为消息发送到 `$file-response/{clientId}` 主题。 这一增强在某些情况下简化了文件传输功能的使用，例如，当客户端使用 MQTTv3 或当代理服务器位于 MQTT 桥接之后时。 更多详情请参阅 [EIP-0021](https://github.com/emqx/eip)。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11541](https://github.com/emqx/emqx/pull/11541) 文件传输能力得到了增强。现在，客户端可以使用异步方式，通过 `$file-async/...` 主题进行文件传输，并通过 `$file-response/{clientId}` 主题订阅命令执行结果。这一改进简化了文件传输功能的使用，尤其适用于 MQTT v3.1/v3.1.1 或使用了 MQTT 桥接的客户端。 更多详情请参阅 [EIP-0021](https://github.com/emqx/eip)。
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 6]
```diff
@@ -1,5 +1,47 @@
...
+
+## 修复
+
+- [#11757](https://github.com/emqx/emqx/pull/11757) 修复了下载不存在的追踪文件时返回的错误响应码。现在，响应码会返回 `404` 而不是 `500`。
+
+- [#11762](https://github.com/emqx/emqx/pull/11762) 修复了 EMQX 中 `built_in_database` 授权数据源的一个问题。通过这次修复，现在在销毁授权数据源时，所有访问控制列表（ACL）记录都会被彻底移除。这解决了以前的问题，即数据库中残留的记录可能在重新创建授权数据源时导致问题。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11762](https://github.com/emqx/emqx/pull/11762) 修复了 EMQX 中 `built_in_database` 授权数据源的一个问题。通过这次修复，现在在删除数据源时，所有 ACL 记录都会被彻底移除。这解决了数据库残留的记录在重新创建授权数据源时仍然存在的问题。
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 7]
```diff
@@ -1,5 +1,47 @@
...
+
+- [#11771](https://github.com/emqx/emqx/pull/11771) 修复了通过 API/Dashboard 进行身份验证管理时 Bcrypt 盐轮次的验证问题。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11771](https://github.com/emqx/emqx/pull/11771) 修复了通过 API/Dashboard 进行身份验证管理时 Bcrypt 盐轮次(salt rounds)的验证问题。
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 8]
```diff
@@ -1,5 +1,47 @@
...
+
+- [#11965](https://github.com/emqx/emqx/pull/11965) 优化了 EMQX 服务的终止过程，确保即使在存在不可用的 MongoDB 资源的情况下，也能够实现优雅停止。
+
+- [#11975](https://github.com/emqx/emqx/pull/11975) 此修复解决了由于对端和服务器同时关闭套接字时发生竞争条件导致的冗余错误日志问题。以前，由操作系统和 EMQX 触发的并发套接字关闭事件会导致不必要的错误记录。通过改进事件处理，本次修复消除了不必要的错误信息。
+
+- [#11987](https://github.com/emqx/emqx/pull/11987) 修复了在尝试设置 TCP/SSL 套接字的 `active_n` 选项时连接崩溃的问题。
+
+  在此修复之前，如果在连接过程中尝试设置 `active_n` 选项时套接字已经关闭，会导致 `case_clause` 崩溃。
+
+- [#11731](https://github.com/emqx/emqx/pull/11731) 将文件传输功能配置添加到了热配置模式。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11731](https://github.com/emqx/emqx/pull/11731) 为文件传输功能添加了热配置支持。
```

File: zh_CN/changes/changes-ee-5.3.md [hunk 9]
```diff
@@ -1,5 +1,47 @@
...
+
+- [#11754](https://github.com/emqx/emqx/pull/11754) 改进了 EMQX 中针对 Postgres 桥接的日志格式化功能。这个改进针对驱动程序返回的错误消息中的 Unicode 字符进行了处理。
```
Comment (wivwiv) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11754](https://github.com/emqx/emqx/pull/11754) 改进了  Postgres 桥接的日志格式化功能，针对驱动程序返回的错误消息中的 Unicode 字符进行了处理。
```

**Review by wivwiv — CHANGES_REQUESTED** [review]

File: zh_CN/changes/changes-ee-5.3.md [hunk 10]
```diff
@@ -1,5 +1,47 @@
...
+
+- [#11975](https://github.com/emqx/emqx/pull/11975) 此修复解决了由于对端和服务器同时关闭套接字时发生竞争条件导致的冗余错误日志问题。以前，由操作系统和 EMQX 触发的并发套接字关闭事件会导致不必要的错误记录。通过改进事件处理，本次修复消除了不必要的错误信息。
+
+- [#11987](https://github.com/emqx/emqx/pull/11987) 修复了在尝试设置 TCP/SSL 套接字的 `active_n` 选项时连接崩溃的问题。
+
+  在此修复之前，如果在连接过程中尝试设置 `active_n` 选项时套接字已经关闭，会导致 `case_clause` 崩溃。
+
+- [#11731](https://github.com/emqx/emqx/pull/11731) 为文件传输功能添加了热配置支持。
+
+- [#11754](https://github.com/emqx/emqx/pull/11754) 改进了  Postgres 桥接的日志格式化功能，针对驱动程序返回的错误消息中的 Unicode 字符进行了处理。
```
Comment (yanzhiemq) [inline zh_CN/changes/changes-ee-5.3.md]:
```suggestion
- [#11754](https://github.com/emqx/emqx/pull/11754) 改进了 Postgres 桥接的日志格式化功能，针对驱动程序返回的错误消息中的 Unicode 字符进行了处理。
```

**Review by zmstone — APPROVED** [review]

**Review by wivwiv — APPROVED** [review]

**Pull request merged**